himl/main.py

```diff
@@ -28,10 +46,25 @@ def do_run(self, opts):
         if opts.output_file is None:
             opts.print_data = True
 
+        merge_list_strategy = ["append"] #default merge strategy for list
+        if opts.merge_list_strategy is not None:
+            if opts.merge_list_strategy[0] == 'default': #default merge strategy provided by himl/deepmerge
+                try:
+                    merge_list_strategy = [DefaultMergeStrategy.from_string(opts.merge_list_strategy[1]).value] #only viable options are append/override/prepend/append_unique
+                except ValueError as err:
+                    print(err)
+                    return       
+            else:
+                imported_module = importlib.import_module(opts.merge_list_strategy[0])
```

Comment:
I don't think importing dynamically an external module is practical for users that are consuming himl as a cli.
Think a user using https://github.com/adobe/himl#using-docker-image

We should remove this option (of providing an external module as a paramter) from this cli and instead allow this only for users consuming himl as a library

Reply:
I agree. Initially I just used main.py as an example to showcase how to use it. If we use main.py as strictly as a cli, it does not make sense. I have removed the option to provide an external module. We will defer documentation for using custom module to README file.

Reply:
@amuraru  @costimuraru Pushed latest code.  Removed dynamic loading of external module, and updated README to describe how to pass custom functions to ConfigProcessor.  Please review. Thanks.